io-ballerina/src/io/character/channel_string_io.bal

```diff
@@ -124,14 +124,14 @@ public function channelWriteString(WritableChannel writableChannel, string conte
     }
 }
 
-# Write an array of lines to a channel.
+# Write an array of lines to a channel
 # ```ballerina
 # string[] content = ["Hello Universe..!!", "How are you?"];
 # io:Error result = io:channelWriteLines(writableChannel, content);
 # ```
-# + writableChannel - A writable channel
-# + content - An array of string lines
-# + return - Either `io:Error` or `()`
+# + writableChannel - A writable channel. The possible inputs are `io:WritableByteChannel` or `io:WritableCharacterChannel`.
+# + content - An array of string lines to write
+# + return - Either an `io:Error` or the null `()` value when the writing was successful
```

Comment:
Same issue as above.

Reply:
Fixed in here https://github.com/ballerina-platform/module-ballerina-io/pull/37/commits/123095180ab4fe7865b5e69d8623895fe6b978e8